artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/AutoCreatedQueueManagerImpl.java

```diff
@@ -46,7 +46,7 @@ public void run() {
                   logger.debug("deleting auto-created queue \"" + queueName + ".\" consumerCount = " + consumerCount + "; messageCount = " + messageCount + "; isAutoDeleteJmsQueues = " + isAutoDeleteJmsQueues);
                }
 
-               server.destroyQueue(queueName, null, false);
```

Comment:
We you could have destroyQueue method the same here?
shouldn't destroyQueue be calling getJMSQueueDeleter().delete(queueName); 

?